admin_guide/scheduling/index.adoc

```diff
@@ -0,0 +1,38 @@
+[[admin-guide-scheduling-index]]
+= Overview
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
+== Overview
+
+Pod scheduling is an internal process that determines how new
+pods are placed onto nodes within the cluster
+
+[[admin-guide-scheduling-index-default]]
+== Default scheduling
+
+{product-title} comes with a default scheduler that should serve the needs of most users. The default scheduler uses both inherent and customizable tools to determine the best fit for a pod. 
+
+For information on how the default scheduler determines pod placement and on customizable parameters, see xref:../../admin_guide/scheduling/scheduler.adoc#admin-guide-scheduler[Default Scheduling].
```

Comment:
Maybe: For information on how the default scheduler determines pod placement and _available_ customizable parameters,